synapse/rest/client/v1/room.py

```diff
@@ -848,7 +849,12 @@ def __init__(self, hs):
 
 
 class RoomAliasListServlet(RestServlet):
-    PATTERNS = client_patterns("/rooms/(?P<room_id>[^/]*)/aliases", unstable=False)
+    PATTERNS = [
+        re.compile(
+            r"^/_matrix/client/unstable/org\.matrix\.msc2432"
+            r"/rooms/(?P<room_id>[^/]*)/aliases"
+        ),
+    ]
```

Comment:
At some point we may want an equivalent `client_patterns` for `/unstable/...`

Reply:
I think `client_patterns` is responsible for a whole bunch of unnecessary clutter in the #5118 department.

What I *would* like to see is putting strings rather than compiled REs into PATTERNs, to save some boilerplate, but I decided to shelve it for now.

Reply:
This is inconsistent with the Client-Server API Spec and API Playground.